src/OpenTelemetry.Exporter.Geneva/GenevaLogExporter.cs

```diff
@@ -443,5 +485,62 @@ private static byte GetSeverityNumber(LogLevel logLevel)
                 return 1;
         }
     }
+
+    private static int SanitizeCategoryName(byte[] buffer, int cursor, ref int validNameLength, string categoryName)
+    {
+        int cursorStartIdx = cursor;
+
+        const int maxStr8LengthInBytes = (1 << 8) - 1;
```

Comment:
Change the access modifier of `MessagePackSerializer.LIMIT_MAX_STR8_LENGTH_IN_BYTES` to reuse that here.